lang/java/reef-tests/src/test/java/org/apache/reef/tests/examples/TestHelloREEF.java

```diff
@@ -48,14 +48,18 @@ public void tearDown() throws Exception {
 
   @Test
   public void testHelloREEF() {
-    final Configuration driverConf = DriverConfiguration.CONF
+    final Configuration driverConf = getDefaultDriverConfiguration();
+    final LauncherStatus state = this.testEnvironment.run(driverConf);
+    Assert.assertTrue("Job state after execution: " + state, state.isSuccess());
+  }
+
+  private Configuration getDefaultDriverConfiguration() {
+    return DriverConfiguration.CONF
         .set(DriverConfiguration.GLOBAL_LIBRARIES, EnvironmentUtils.getClassLocation(this.getClass()))
```

Comment:
Nit: seems there is no need to change this file.

Reply:
Correct. Will remove this file from the changeset.

---
In reply to: [201770287](https://github.com/sharathmalladi/reef/pull/5#discussion_r201770287) [](ancestors = 201770287)